src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/Controllers/WeatherForecastController.cs

```diff
@@ -100,8 +100,10 @@ public WeatherForecastController(ILogger<WeatherForecastController> logger)
         [HttpGet]
         public IEnumerable<WeatherForecast> Get()
         {
+#if (!NoAuth)
```

Comment:
@jmprieur FYI I saw some of our test fail and I'm fairly certain the call to `VerifyUserHasAnyAcceptedScope` should be skipped for no Auth. Do you agree?